server/channels/app/notification.go

```diff
@@ -1254,6 +1254,18 @@ func (a *App) getGroupsAllowedForReferenceInChannel(channel *model.Channel, team
 				groupsMap[*group.Group.Name] = &group.Group
 			}
 		}
+
+		opts.Source = model.GroupSourceCustom
+		var customgroups []*model.Group
+		customgroups, err = a.Srv().Store().Group().GetGroups(0, 0, opts, nil)
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to get custom groups")
+		}
+		for _, group := range customgroups {
+			if group.Name != nil {
+				groupsMap[*group.Name] = group
+			}
+		}
```

Comment:
Can a customer group have the same name as a channel or team group, i.e. are group names globally unique?

Reply:
Sorry missed this comment, the group names are globally unique